packages/amplify-category-notifications/src/channel-apns.ts

```diff
@@ -73,14 +61,10 @@ export const enable = async (context: $TSContext, successMessage: string | undef
     if (context.exeInfo.serviceMeta.output[channelName]) {
       channelOutput = context.exeInfo.serviceMeta.output[channelName];
     }
-    const question: QuestionCollection<{ [x: string]: unknown; }> = {
-      name: 'DefaultAuthenticationMethod',
-      type: 'list',
-      message: 'Choose authentication method used for APNs',
-      choices: ['Certificate', 'Key'],
-      default: channelOutput.DefaultAuthenticationMethod || 'Certificate',
+    const authMethod = await prompter.pick('Select the authentication method for the APNS channel', ['Certificate', 'Key'], channelOutput.DefaultAuthenticationMethod || 'Certificate');
```

Comment:
I think this needs to be:
```suggestion
    const authMethod = await prompter.pick('Select the authentication method for the APNS channel', ['Certificate', 'Key'], {initial: byValue(channelOutput.DefaultAuthenticationMethod || 'Certificate')});
```